scs-app/src/styles.scss

```diff
@@ -1 +1,4 @@
 /* You can add global styles to this file, and also import other style files */
+
+/* Importing Bootstrap SCSS file. */
+@import '~bootstrap/scss/bootstrap';
```

Comment:
I think we already have this imported -- is this causing issues using boostrap if we don't have this?

Reply:
ng-bootstrap must've added it during install, took it out!